### PR TITLE
Add example for amazon.aws.aws_secret with region and aws_profile parameters

### LINKED ISSUE #416: Add new example for `amazon.aws.aws_secret` that includes use of `region=` and `aws_profile=` parameters
### Summary

In order to get my `amazon.aws.aws_secret` lookup to work, I had to add the `region=` and `aws_profile=` parameters. I found these through Google searches and guesswork... Would it be worth having a new example that shows them in use?

For example, to take one of the existing examples and modify it a little:
```
- name: lookup secretsmanager secret in the current region using the nested feature - specify region and AWS Profile
  debug: msg="{{ lookup('amazon.aws.aws_secret', 'secrets.environments.production.password', nested=true, region=us-east-1, aws_profile=dev-profile) }}"
  # The secret can be queried using the following syntax: `aws_secret_object_name.key1.key2.key3`.
  # If an object is of the form `{"key1":{"key2":{"key3":1}}}` the query would return the value `1`.
  # region= should be changed to reflect what region AWS Secret is stored
  # aws_profile= should reflect what AWS Profile to use, that has access to the AWS Secret
```
Just a thought, if you feel this isn't needed, please ignore.

Thanks!

### Issue Type

Documentation Report

### Component Name

amazon.aws.aws_secret

### Ansible Version

```console (paste below)
$ ansible --version

```

### Collection Versions

```console (paste below)
$ ansible-galaxy collection list
```

### Configuration

```console (paste below)
$ ansible-config dump --only-changed

```

### OS / Environment

_No response_

### Additional Information

_No response_

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by mandar242**
##### SUMMARY

Added a new example for amazon.aws.aws_secret that includes use of the region and aws_profile parameters.

Resolves #416.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

amazon.aws.aws_secret

### CODE REVIEW

**Comment by jillr** [comment]
recheck

**Comment by tremble** [comment]
recheck

File: plugins/lookup/aws_secret.py [hunk 1]
```diff
@@ -95,6 +95,14 @@
    debug: msg="{{ lookup('amazon.aws.aws_secret', 'secrets.environments.production.password', nested=true) }}"
    # The secret can be queried using the following syntax: `aws_secret_object_name.key1.key2.key3`.
    # If an object is of the form `{"key1":{"key2":{"key3":1}}}` the query would return the value `1`.
+ - name: lookup secretsmanager secret in a specific region using specified region and aws profile using nested feature
+   debug: >
+    msg="{{ lookup('amazon.aws.aws_secret', 'secrets.environments.production.password', region=region, aws_profile=aws_profile,
+    aws_access_key=aws_access_key, aws_secret_key=aws_secret_key, nested=true) }}"
+   # The secret can be queried using the following syntax: `aws_secret_object_name.key1.key2.key3`.
+   # If an object is of the form `{"key1":{"key2":{"key3":1}}}` the query would return the value `1`.
+   # Region is the aws region the aws secret is stored.
```
Comment (Akasurde) [inline plugins/lookup/aws_secret.py:104]:
```suggestion
   # Region is the AWS region where the AWS secret is stored.
```

**Review by tremble — APPROVED** [review]

**Comment by tremble** [comment]
recheck

**Comment by tremble** [comment]
recheck

**Pull request merged**